fec/fec/static/js/calendar.js

```diff
@@ -339,6 +348,27 @@ Calendar.prototype.handleEventClick = function(calEvent, jsEvent, view) {
   }
 };
 
+Calendar.prototype.handleKeypress = function(e) {
```

Comment:
Sure, that seems better. A docstring or comment might be useful here too for future selves.